ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml

```diff
@@ -268,7 +268,7 @@
     launch_file_input: "{{ launch_file_input_gui_right_realhw_8DOF }}"
     desktop_icon_name: "4 - Launch Right Teleop GUI 8DOF"
     desktop_icon_path: "Shadow Advanced Launchers/4 - Launch Right Teleop GUI 8DOF"
-  when: bimanual|bool or hand_side=="right" and arm_servo|bool == true
+  when: (bimanual|bool or hand_side=="right") and (arm_servo|bool)
```

Comment:
I think the convention we use is 
- condition1
- condition2

Reply:
We use both, either one is fine, I think we mainly use this for 3+ conditionals.